modules/nf-core/gatk4/haplotypecaller/gatk4-haplotypecaller.diff

```diff
@@ -9,14 +9,5 @@ Changes in module 'nf-core/gatk4/haplotypecaller'
  
      conda "bioconda::gatk4=4.3.0.0"
      container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-@@ -36,7 +36,7 @@
-     if (!task.memory) {
-         log.info '[GATK HaplotypeCaller] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
-     } else {
--        avail_mem = task.memory.giga
-+        avail_mem = task.memory.giga - 1
```

Comment:
This doens't seem right?
-> restore the module to default, then change to `process_single`, then generate a patch through `nf-core patch`

Reply:
That was the old patch, this part has been deleted and only the label change is in the patch

Reply:
ah, carry on then!

Reply:
Thanks! Let's wait for the tests to complete :p